services/ui-src/src/components/pages/Export/ExportedReportPage.tsx

```diff
@@ -30,6 +31,8 @@ export const ExportedReportPage = () => {
   const { exportVerbiage } = getReportVerbiage(reportType);
   const { metadata, reportPage, tableHeaders } = exportVerbiage;
 
+  const Helmet = HelmetImport as ComponentClass<HelmetProps>;
```

Comment:
copied from HCBS. honestly not sure why the old way doesn't work because the Helmet docs still use the old way
https://github.com/Enterprise-CMCS/macpro-mdct-hcbs/pull/132/files#diff-b49fbeafbfa777864db8db0f3a21830f3731c208fb897c6ebcc95fb0a5a019eb